tools/cloud-build/daily-tests/tests/batch-mpi.yml

```diff
@@ -0,0 +1,27 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+---
+test_name: batch-mpi
+deployment_name: batch-mpi-{{ build }}
+zone: us-central1-c
+workspace: /workspace
+blueprint_yaml: "{{ workspace }}/examples/batch-mpi.yaml"
+network: "default"
+remote_node: "{{ deployment_name }}-batch-login"
+post_deploy_tests:
+- test-batch-submission.yml
```

Comment:
it might be useful for this task to print its job UUID or name or something else to assist finding it in Cloud Logging. I tracked down the related logs and they all looked happy.

Reply:
Please create a task for that unless you object.

Reply:
I agree with the idea. I almost just want to add here but don't want to risk introducing a but into the test and would rather do in a follow on pr. I have opened a task.